src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -270,11 +276,12 @@ def as_markdown(self):
             location='{}:{}:{}'.format(self.path, self.line, self.char),
             body=self.body,
             check=self.check,
-            in_patch=self.in_patch and 'yes' or 'no',
+            in_patch=self.in_patch() and 'yes' or 'no',
```

Comment:
Nit: same nit here and for is_new a few lines after (inline if is more readable)